src/Dictionaries/CassandraDictionarySource.cpp

```diff
@@ -17,12 +18,15 @@ void registerDictionarySourceCassandra(DictionarySourceFactory & factory)
                                    [[maybe_unused]] const Poco::Util::AbstractConfiguration & config,
                                    [[maybe_unused]] const std::string & config_prefix,
                                    [[maybe_unused]] Block & sample_block,
-                                                    ContextPtr /* global_context */,
+                                                    ContextPtr global_context,
```

Comment:
Maybe this will fix build error?
```suggestion
                                   [[maybe_unused]] ContextPtr global_context,
```